terraform/modules/benchmark/service/run_benchmark.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'ruby_common/slack_client'
+require_relative 'ruby_common/utils'
+
+require 'logger'
+require 'fileutils'
+
+# Retrieves an environmental variable, failing if its not set or empty.
+def get_and_assert_env_variable(name)
+  var = ENV.fetch(name, nil)
+  raise "Please set #{name} environmental variable" if var.nil? || var.empty?
+
+  var
+end
+
+SLACK_TOKEN = get_and_assert_env_variable 'SLACK_API_TOKEN'
+CHANNEL = get_and_assert_env_variable 'SLACK_NOTIF_CHANNEL'
+SCRIPTS_DIR = get_and_assert_env_variable 'BASE_FOLDER'
+LOG_DIR = get_and_assert_env_variable 'BASE_FOLDER'
+
+loop do
+  # Current datetime, to append to the log files
+  datetime = Time.new.strftime '%FT%H:%M:%S'
+  health_log = "#{LOG_DIR}/benchmark_#{datetime}_health"
+  sync_log = "#{LOG_DIR}/benchmark_#{datetime}_sync"
```

Comment:
What is sync and health and sync log exactly and how are they created / populated?

Reply:
fixed, now there is only one log file `benchmark_#{datetime}_run`